src/rpc/server.ts

```diff
@@ -57,12 +57,12 @@ export class MuRPCServer<Protocol extends MuRPCProtocol<any>, Connection extends
                 } catch (e) {
                     logger && logger.error(e);
                     response.type = 'error';
-                    if (e instanceof Error && e.stack) {
-                        response.data = `Internal error:\n${e.stack}`;
+                    if (e instanceof Error && e.message) {
```

Comment:
we could probably skip the instanceof check? maybe { message, stack }? @mikolalysenko

Reply:
yeah